Crust/Mapper/MappingOperator.swift

```diff
@@ -38,6 +47,9 @@ public func <- <T: Mappable, U: Mapping, C: MappingContext where U.MappedObject
     return mapField(&field, map: map)
 }
 
+public typealias MappableAndJSONable = protocol<JSONable, Mappable>
```

Comment:
I don't see this being used at all. If you don't plan on using it please delete it. Otherwise I'd prefer the name `MappableJSONable` and then please move to MappingProtocols.Swift file.